x/pylons/client/cli/tx_cookbook.go

```diff
@@ -52,21 +52,30 @@ pylonsd tx pylons create-cookbook "loud123456" "Legend of the Undead Dragon" "Co
 				return sdkerrors.Wrap(sdkerrors.ErrInvalidRequest, err.Error())
 			}
 
-			clientCtx, err := client.GetClientTxContext(cmd)
-			if err != nil {
-				return err
+			var clientCtx client.Context
+
+			altCtx := GetAltenativeContext()
```

Comment:
misspelling for "GetAlternativeContext", this is in multiple places.

Reply:
whoops!